pkg/kubelet/secret/secret_manager.go

```diff
@@ -111,6 +112,19 @@ const (
 	defaultTTL = time.Minute
 )
 
+func getTPClient(kubeClients []clientset.Interface, tenant string) clientset.Interface {
```

Comment:
Remember to add one line to handle the case of one TP, for the purpose of M1 testing.

Reply:
next PR will be for this